model/ResultsService.php

```diff
@@ -396,6 +397,34 @@ public function getItemFromItemResult($itemCallId, $itemVariables = [])
         return $item;
     }
 
+    /**
+     * @deprecated temp bugfix for remote deliveries
```

Comment:
Could you refer to the proper solution while adding something deprecated?

Reply:
This should be a TODO with a follow up task IMO.

Reply:
may be it's not correct to mark it as deprecated. I just want to note that it's temp solution which waiting development of new features for this use case. May be you can suggest better way for this note, or may be it's shouldn't be noted?

Reply:
`TODO` sounds good. I will change it now